README.md

```diff
@@ -103,13 +87,10 @@ The extension does most of the heavy lifting when it comes to gathering and prep
 
 Depending on the extension version you are using, you could have a different PHP API client version powering the extension's backend functionality.
 
-| Extension Version | API Client Version                                                        |
-|-------------------|---------------------------------------------------------------------------|
-| v1.x              | [1.28.0](https://github.com/algolia/algoliasearch-client-php/tree/1.28.0) |
-| v2.x              | [2.5.1](https://github.com/algolia/algoliasearch-client-php/tree/2.5.1)   |
-| v3.x              | [2.5.1](https://github.com/algolia/algoliasearch-client-php/tree/2.5.1)   |
-| v3.6.x            | [3.2.0](https://github.com/algolia/algoliasearch-client-php/tree/3.2.0)   |
-| >= v3.11.0        | [3.3.2](https://github.com/algolia/algoliasearch-client-php/tree/3.3.2)   |
+| Extension Version | API Client Version     | PHP              | Magento   |
+|-------------------|------------------------|------------------|-----------|
+| v3.13.x           | [3.3.2](https://github.com/algolia/algoliasearch-client-php/tree/3.3.2)           | `^7.2 \|\| ^8.0` | `2.4.x` |           
+| v3.14.x           | `4.0.x`                | `>=8.1`          | `2.4.x`   |
```

Comment:
I think this is ok to let the older versions and specify that we don't support them anymore, this would be easier for the users which are on an obsolete version (see CR-6585 for example) and also for us to track more easily the compatibiliy changes, WDYT ?

Reply:
@damcou Version 1 and 2 are VERY old - going on 4 years now. But we shouldn't remove v3.x to 3.11.x. 

I also don't think it makes sense to include the PHP or Magento versions here. This is just providing verrsioning info for the PHP API client. Let's move them instead under the "Magento 2.4 compatibility" section. 

@mrahman3177 Can you please make the requested changes?

Reply:
Totally agreed with your opinion with versions 1 and 2, I should have been more precise ;)